orderer/common/cluster/block_fetcher_test.go

```diff
@@ -57,8 +57,8 @@ func TestBlockFetcherHappyPath(t *testing.T) {
 		},
```

Comment:
1. Can we put `t.Parallel()` everywhere so the tests run in parallel? 
2. Please remove `fmt.Println()` from the code